crates/rolldown/src/bundler.rs

```diff
@@ -40,31 +40,39 @@ impl Bundler {
 impl Bundler {
   #[tracing::instrument(level = "debug", skip_all)]
   pub async fn write(&mut self) -> Result<BundleOutput> {
-    let dir = self.options.cwd.as_path().join(&self.options.dir).to_string_lossy().to_string();
-
     let mut output = self.bundle_up(true).await?;
-
     self.plugin_driver.write_bundle(&mut output.assets).await?;
 
-    self.fs.create_dir_all(dir.as_path()).map_err(|err| {
-      anyhow::anyhow!(
-        "Could not create directory for output chunks: {:?} \ncwd: {}",
-        dir.as_path(),
-        self.options.cwd.display()
-      )
-      .context(err)
-    })?;
-    for chunk in &output.assets {
-      let dest = dir.as_path().join(chunk.filename());
-      if let Some(p) = dest.parent() {
-        if !self.fs.exists(p) {
-          self.fs.create_dir_all(p).unwrap();
-        }
-      };
-      self.fs.write(dest.as_path(), chunk.content_as_bytes()).map_err(|err| {
-        anyhow::anyhow!("Failed to write file in {:?}", dir.as_path().join(chunk.filename()))
-          .context(err)
+    if let Some(dest) = &self.options.file {
```

Comment:
Feel like there will be problems in this implementation for working with asset modules and future copy and file loader #1295.

---

I just give it a rough thinking and it's not easy job to make this feature right. I'm so far too busy to give detailed instructions on how to support this feature.